src/tools/sourcecred.js

```diff
@@ -0,0 +1,134 @@
+//@flow
+
+/*
+ * CLI for loading SourceCred graphs. Main entry point for SourceCred users.
+ *
+ * For usage, run node bin/sourcecred.js --help
+ */
+
+import yargs from "yargs/yargs";
+import os from "os";
+import path from "path";
+
+function pluginGraph(
+  repoOwner: string,
+  repoName: string,
+  plugin: string,
+  githubToken?: string
+) {
+  console.log(
+    `
+    pluginGraph
+    repoOwner: ${repoOwner}
+    repoName: ${repoName}
+    plugin: ${plugin}
+    githubToken?: ${githubToken || "null"}
+    `
+  );
+}
+
+function graph(
+  repoOwner: string,
+  repoName: string,
+  outputDir: string,
+  githubToken: string
+) {
+  console.log(
+    `graph
+    repoOwner: ${repoOwner}
+    repoName: ${repoName}
+    outputDir: ${outputDir}
+    githubToken: ${githubToken}`
+  );
+}
+
+function combine(files: string[]) {
+  console.log(`combine files: ${files.join(",")}`);
+}
+
+function main() {
+  function defaultBuilder(yargs) {
+    return yargs.strict();
+  }
+
+  function repoPositional(yargs) {
+    return yargs
+      .positional("repoOwner", {
+        describe: "the owner of the repository",
+      })
+      .positional("repoName", {describe: "the name of the repository"});
+  }
+
+  function githubToken(yargs, options) {
+    const required = options != null && options.required;
+    yargs = yargs.option("github-token", {
+      demandOption: required,
+      describe:
+        "a GitHub api token, as generated at https://github.com/settings/tokens/new",
```

Comment:
nit: Can we always write "API" (capitalized), please?

Reply:
Done